tlapm.opam

```diff
@@ -0,0 +1,50 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "TLA+ Proof Manager"
+description: "TLA+ Proof Manager"
+maintainer: [
+  "Kaustuv Chaudhuri"
+  "Denis Cousineau"
+  "Damien Doligez"
+  "Leslie Lamport"
+  "Tomer Libal"
+  "Stephan Merz"
+  "Jean-Baptiste Tristan"
+  "Hernan Vanzetto"
+]
+authors: [
+  "Kaustuv Chaudhuri"
+  "Denis Cousineau"
+  "Damien Doligez"
+  "Leslie Lamport"
+  "Tomer Libal"
+  "Stephan Merz"
+  "Jean-Baptiste Tristan"
+  "Hernan Vanzetto"
```

Comment:
again, needs updating

Reply:
This file is generated from `dune-project`; it will be enough to update these lists in a single place.